Apps.DeepL/Actions/GetUsageAction.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Apps.DeepL.Requests;
+using Apps.DeepL.Responses;
+using Blackbird.Applications.Sdk.Common;
+using Blackbird.Applications.Sdk.Common.Actions;
+using Blackbird.Applications.Sdk.Common.Invocation;
+using Blackbird.Applications.SDK.Extensions.FileManagement.Interfaces;
+using DeepL;
+
+namespace Apps.DeepL.Actions;
+
+[ActionList]
+public class GetUsageActionActions : DeepLInvocable
+{
+
+    public GetUsageActionActions(InvocationContext invocationContext) : base( invocationContext )
+    {
+
+    }
+
+    [Action("GetUsage", Description = "Get usage information")]
+    public async Task<GetUsageResponse> GetUsage()
+    {
+        var result = await Client.GetUsageAsync();
+        var response = new GetUsageResponse();
+        if (result.Character != null)
```

Comment:
No need to check each property for null and then add it to the response. You can just add all the values with nullchecks, it will be null anyway